src/images/pc-svg-1.svg

```diff
@@ -0,0 +1,68 @@
+<?xml version="1.0" encoding="iso-8859-1"?>
+<!-- Generator: Adobe Illustrator 19.0.0, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->
+<svg version="1.1" id="Layer_1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px"
+	 viewBox="0 0 512.024 512.024" style="enable-background:new 0 0 512.024 512.024;" xml:space="preserve">
+<path d="M359.548,485.792H152.476c-4.328,0-7.864-3.536-7.864-7.864V7.864c0-4.328,3.536-7.864,7.864-7.864h207.072
+	c4.328,0,7.864,3.536,7.864,7.864v470.064C367.412,482.248,363.876,485.792,359.548,485.792z"/>
+<path style="fill:#333333;" d="M155.756,0v148.096c0,11.536,9.44,20.968,20.968,20.968h158.584c11.536,0,20.968-9.44,20.968-20.968
+	V0H155.756z"/>
+<rect x="155.756" y="74.264" width="200.56" height="5.24"/>
+<rect x="155.756" y="10.048" width="200.56" height="5.24"/>
+<rect x="218.644" y="125.376" style="fill:#5B5C5F;" width="19.744" height="5.24"/>
+<rect x="191.444" y="125.376" style="fill:#DB2B42;" width="19.744" height="5.24"/>
+<rect x="155.972" y="482.32" width="200.08" height="29.704"/>
+<rect x="300.268" y="125.376" style="fill:#DB2B42;" width="19.744" height="5.24"/>
+<rect x="273.052" y="125.376" style="fill:#5B5C5F;" width="19.744" height="5.24"/>
+<rect x="245.852" y="125.376" style="fill:#DB2B42;" width="19.744" height="5.24"/>
+<circle style="fill:#5B5C5F;" cx="327.22" cy="210.568" r="21.152"/>
+<path style="fill:#999999;" d="M327.22,233.056c-12.392,0-22.488-10.096-22.488-22.488c0-12.392,10.096-22.488,22.488-22.488
+	c12.392,0,22.488,10.096,22.488,22.488S339.612,233.056,327.22,233.056z M327.22,190.744c-10.928,0-19.824,8.896-19.824,19.824
+	c0,10.928,8.896,19.824,19.824,19.824c10.928,0,19.824-8.896,19.824-19.824S338.148,190.744,327.22,190.744z"/>
+<circle cx="327.22" cy="210.568" r="15.024"/>
+<circle style="fill:#DB2B42;" cx="327.22" cy="210.568" r="6.392"/>
+<g style="opacity:0.5;">
+	<rect x="222.812" y="421.568" style="fill:#FFFFFF;" width="66.4" height="20.096"/>
+</g>
+<path style="fill:#5B5C5F;" d="M320.668,142.352H191.356l-18.32,26.416c1.2,0.176,2.424,0.304,3.688,0.304h158.584
+	c1.264,0,2.488-0.128,3.688-0.304L320.668,142.352z"/>
+<circle cx="269.532" cy="155.704" r="3.928"/>
+<circle cx="283.388" cy="155.704" r="3.928"/>
+<circle cx="297.212" cy="155.704" r="3.928"/>
+<circle cx="311.1" cy="155.704" r="3.928"/>
+<g style="opacity:0.25;">
+	<path style="fill:#FFFFFF;" d="M356.924,0H256.012v485.792h100.912c5.768,0,10.488-4.72,10.488-10.488V10.488
+		C367.412,4.72,362.692,0,356.924,0z"/>
+</g>
+<rect x="196.684" y="153.04" width="24.24" height="5.336"/>
+<rect x="231.764" y="153.04" width="24.24" height="5.336"/>
+<g>
+</g>
+<g>
+</g>
+<g>
```

Comment:
Try deleting these extra `g` tag and see if its changing the SVG. If not, then you can delete them.

Reply:
Okay, I'll do that. Thanks, Apurva